config/src/inline/conf_parser.rs

```diff
@@ -0,0 +1,205 @@
+use regex::Regex;
+
+use crate::{InlineConfigError, NatSpec};
+
+use super::{remove_whitespaces, INLINE_CONFIG_PREFIX};
+
+/// Errors returned by the [`InlineConfigParser`] trait.
+#[derive(Debug, Clone, PartialEq, Eq, thiserror::Error)]
+pub enum InlineConfigParserError {
+    /// An invalid configuration property has been provided.
+    /// The property cannot be mapped to the configuration object
+    #[error("'{0}' is an Invalid config property")]
```

Comment:
This was recommended by @mds1.  The goal is to give emphasis to the word **Invalid**

Reply:
Previously when “invalid” was the first word I suggested it be capitalized, but since it’s no longer the first word I agree with @Evalir’s suggestion here. To clarify, the suggestion was just fixing the casing, not around emphasis 😅